bundles/org.openhab.binding.ecovacs/src/main/java/org/openhab/binding/ecovacs/internal/EcovacsBindingConstants.java

```diff
@@ -93,8 +94,11 @@ public class EcovacsBindingConstants {
             new StateOptionEntry<>(CleanMode.SPOT_AREA, "spotArea", DeviceCapability.SPOT_AREA_CLEANING),
             new StateOptionEntry<>(CleanMode.CUSTOM_AREA, "customArea", DeviceCapability.CUSTOM_AREA_CLEANING),
             new StateOptionEntry<>(CleanMode.SINGLE_ROOM, "singleRoom", DeviceCapability.SINGLE_ROOM_CLEANING),
-            new StateOptionEntry<>(CleanMode.PAUSE, "pause"), new StateOptionEntry<>(CleanMode.STOP, "stop"),
-            new StateOptionEntry<>(CleanMode.WASHING, "washing"), new StateOptionEntry<>(CleanMode.DRYING, "drying"),
+            new StateOptionEntry<>(CleanMode.SCENE_CLEAN, "sceneClean", DeviceCapability.SCENARIO_CLEANING),
+            new StateOptionEntry<>(CleanMode.PAUSE, "pause"), //
+            new StateOptionEntry<>(CleanMode.STOP, "stop"), new StateOptionEntry<>(CleanMode.AUTO_EMPTY, "autoEmpty"),
```

Comment:
Why not enforce the line break here as well?

(NB: I find the 'need to fill the line' rule highly stupid for cases like this one.)

Reply:
Good spotting, I missed them!

Reply:
fixed